language/stdlib/modules/genesis.move

```diff
@@ -91,6 +91,9 @@ module Genesis {
         Libra::grant_burn_capability_for_sender<Coin1::T>();
         Libra::grant_burn_capability_for_sender<Coin2::T>();
         Libra::grant_burn_capability_for_sender<LBR::T>();
+        Libra::grant_mint_capability<Coin1::T>();
+        Libra::grant_mint_capability<Coin2::T>();
+        Libra::grant_mint_capability<LBR::T>();
```

Comment:
@dmitri-perelman / @sblackshear do we want LBR mint capability for treasure/compliance account?

Reply:
Good call, we don't want to mint LBR in general in prod, so no need to have this capability.

Reply:
Agreed--the `MintCapability` for LBR should live in `LBR::Reserve` and nowhere else.

Reply:
We probably should remove burn capability for LBR as well from TC account?
@moeziniaf since you are here, do you mind removing it? (line `grant_burn_capability_for_sender<LBR::T>`)